pup-0006.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Commit Bit Assignment
+author: Ina Panova
+created: 15-08-2018
+status: Approved
+version: 1
+---
+
+## Summary
+
+For [Pulp project repositories](https://github.com/pulp/), there is no written process to describe
+giving the commit bit to a contributor. The existing committers want to openly document the process.
+
+
+## Motivation
+
+Comitter - is an individual who has write permissions to a
+[Pulp Organization](https://github.com/pulp/) repository.
+To have "a commit bit" means that one is permitted to commit those changes. 
+
+Historically the commit bit was given on day 0 when hired on the core team at Red Hat.  In Oct 2017
+the existing committers decided to stop that practice and instead document an open process.
+Engineers hired on the Pulp team since Oct '17 have not received commit bit.
+
+We have not yet documented an open process of which to become a committer.
+The process outlined in this document is inspired by processes in Chromium projects and Apache
+Software Foundation.
+
+
+## Scope
+
+This policy is intended as the default for all software in the
+[Pulp Organization](https://github.com/pulp/) unless explicitly stated otherwise.
+
+
+## Becoming a committer.
+
+Overall the candidate must have demonstrated:
+* project commitment, active involvement and care towards the needs of all Pulp users and not only
+  their own interests
+* to provide constructive code reviews
```

Comment:
for language reasons I think this should be s/to provide/contribution of/  This makes the above sentence 'must have demonstrated contribution of ...' similar to the others.